src/software/amazon/ion/UnknownSymbolException.java

```diff
@@ -29,10 +29,17 @@
     private static final long serialVersionUID = 1L;
 
     private final int mySid;
+    private final String myText;
 
     public UnknownSymbolException(int sid)
     {
         mySid = sid;
+        myText = null;
+    }
+    public UnknownSymbolException(String text)
```

Comment:
I'd change the parameter to 'message' and in `getMessage` just return it as-is. Because what's being passed in isn't actually symbol text. This constructor should be used when an unknown symbol causes a problem, but we don't know its symbol ID (as in the case of structs with unknown field names).

Reply:
What symbol would it be the text for? If you look at the only place you're actually using it, it's just a message.